addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -78,3 +83,218 @@ def _get_default_payment_method_id(self):
+        return True
+
+    def _create_webhook(self):
+        """Create a webhook using the Stripe API
+        """
+        if self.stripe_webhook_secret:
+            return {
+                'type': 'ir.actions.client',
+                'tag': 'display_notification',
+                'params': {
+                    'message': _('You already have defined a Stripe Webhook.'),
+                    'sticky': False,
+                    'type': 'danger',
+                }
+            }
+        if 'localhost' in self.company_id.get_base_url():
+            return {
+                'type': 'ir.actions.client',
+                'tag': 'display_notification',
+                'params': {
+                    'message': _(
+                        'Your URL should be publicly available in order to create a Stripe Webhook.'
+                    ),
+                    'sticky': False,
+                    'type': 'danger',
+                }
+            }
+        webhook = self._stripe_make_request(
+            'webhook_endpoints',
+            payload={
+                'url': self._get_webhook_url(),
+                'enabled_events[]': WEBHOOK_HANDLED_EVENTS,
+                'api_version': API_VERSION,
+            }
+        )
+        self.stripe_webhook_secret = webhook.get('secret')
+
+    # -------------------------------------------
+    # Stripe Connect Onboarding
+    # -------------------------------------------
+    def action_stripe_onboarding_account(self, onboarding='payment'):
+        """Action to start Stripe Onboarding
+
+        This method creates the Stripe account and generates an account link to start Stripe Connect
+        Onboarding. If it fails to create one, return to the payment acquirer form.
+
+        :returns: an action
+        :rtype: dict
+        """
+        self.ensure_one()
+        if self.state == 'enabled':
+            return {'type': 'ir.actions.act_window_close'}
+        # Account creation
+        account_id = self._stripe_connect_account_id()
+        # Link generation
+        stripe_onboarding_url = self._stripe_connect_link(account_id, onboarding)
+        if stripe_onboarding_url:
+            return {
+                'type': 'ir.actions.act_url',
+                'url': stripe_onboarding_url,
+                'target': 'self',
+            }
+        action = self.env["ir.actions.actions"]._for_xml_id("payment.action_payment_acquirer")
+        action.update({
+            'views': [[False, 'form']],
+            'res_id': self.id,
+        })
+        return action
+
+    def _stripe_connect_account_id(self):
+        """Gets Stripe Account Id
+
+        This method use the Stripe API to create a new account and get its account id.
+
+        :returns: a Stripe Account Id
+        :rtype: str
+        """
+        account = self._stripe_call_proxy(
+            'accounts', payload=self._stripe_connect_create_account_payload()
+        )
+        return account.get('id')
```

Comment:
Then, when we'll store the stripe account id in master, we'll have to make a request to Stripe to retrieve the account and just take the stripe account id in consideration ? It seems to me that this may lead to additional overhead, doesn't it ?

Reply:
I'm not sure to understand the issue here... Why would we store it in master?

Reply:
After checking the saas module, it appears that you do need to return an account and not just its id!